.github/actions/build-prqlc/action.yaml

```diff
@@ -43,15 +43,15 @@ runs:
     - if: runner.os == 'Windows' && inputs.profile == 'release'
       shell: bash
       run: |
-        echo 'RUSTFLAGS=-Ctarget-feature=+crt-static' >>"$GITHUB_ENV"
+        echo 'RUSTFLAGS=-Ctarget-feature=+crt-static' >> "$GITHUB_ENV"
```

Comment:
Why was this change made?

Reply:
Ah sorry, I hadn't realized the existing one was the standard approach — we can revert...

Reply:
I see. Never mind. It doesn't make much difference.

Reply:
May as well! I put in #3326...